pom.xml

```diff
@@ -14,15 +14,16 @@
         <grpc.version>1.29.0</grpc.version>
         <hamcrest.version>1.3</hamcrest.version>
         <java.version>1.8</java.version>
-        <junit.version>4.12</junit.version>
+        <json.version>20200518</json.version>
+        <junit.version>4.13.1</junit.version>
         <karaf.version>4.1.5</karaf.version>
         <log4j.version>2.13.3</log4j.version>
         <metrics.version>4.1.2</metrics.version>
         <mockito.version>2.18.0</mockito.version>
         <okio.bundle.version>1.14.0_1</okio.bundle.version>
         <okhttp.version>3.10.0</okhttp.version>
         <okhttp.bundle.version>3.10.0_2</okhttp.bundle.version>
-        <opennms.api.version>0.5.1-SNAPSHOT</opennms.api.version>
+        <opennms.api.version>0.5.2-SNAPSHOT</opennms.api.version>
```

Comment:
@j-white I had to bump the version in order to get the Integration tests running. I am not sure of the implications:
The requirement only affects the tests but it asks now for a higher version not necessarily provided by the Opennms installation?
What do you think we should do?